frontend/src/index.css

```diff
@@ -0,0 +1,35 @@
+.menu {
```

Comment:
If you have to use an external stylesheet for the menu, rename this file to `menu.css`